app/categorization/llm_helper.py

```diff
@@ -0,0 +1,127 @@
+from datetime import datetime
+import json
+from typing import Dict, Any, Union
+from langchain_community.chat_models import ChatOllama
+from langchain_core.prompts import PromptTemplate
+
+
+def SexLevel(result_dict: Dict[str, str], r: str, key: str) -> Dict[str, Any]:
+    value = result_dict[key]
+    value = value.casefold()
+    var1: str = ""
+    var2: str = ""
+    var3: str = ""
+
+    if "1" in value or "2" in value or "3" in value:
+        var1 = "1"
+        var2 = "2"
+        var3 = "3"
+    elif "0" in value or "1" in value or "2" in value:
+        var1 = "0"
+        var2 = "1"
+        var3 = "2"
```

Comment:
I think this here might contradict each other, because both ifs have `1` and `2` included - maybe we can discuss it in the meeting which one we should use - I would suggest the 0,1,2 option since I think this is the most common.